### DISCUSSION TITLE
For non-spherical bodies, replace `radius` and `size` with `width` and `height`

### DISCUSSION THREAD

**Discussion by offwa** [brandon-rhodes/pyephem#276] [Ideas]
Self explanatory

**Comment by brandon-rhodes** [comment]
The underlying C library that PyEphem is a wrapper around doesn't supply those concepts, so I don't think they're something that can be added to PyEphem itself? If someone finds them in the code, please comment here and point them out, and we can make a plan for adding them. Thanks!
